yarn-project/end-to-end/src/e2e_p2p/slashing.test.ts

```diff
@@ -90,13 +94,22 @@ describe('e2e_p2p_slashing', () => {
       client: t.ctx.deployL1ContractsValues.publicClient,
     });
 
-    const slashingInfo = async () => {
-      const bn = await t.ctx.cheatCodes.eth.blockNumber();
+    const getRoundAndSlotNumber = async () => {
       const slotNumber = await rollup.getSlotNumber();
-      const roundNumber = await slashingProposer.read.computeRound([slotNumber]);
+      return { roundNumber: await slashingProposer.read.computeRound([slotNumber]), slotNumber };
+    };
+
+    /**
+     * Get the slashing info for a given round number.
+     * @param roundNumber - The round number to get the slashing info for.
+     * @returns The current block number, current slot number and the slashing info.
+     */
+    const slashingInfo = async (roundNumber: bigint) => {
       const instanceAddress = t.ctx.deployL1ContractsValues.l1ContractAddresses.rollupAddress.toString();
       const info = await slashingProposer.read.rounds([instanceAddress, roundNumber]);
       const leaderVotes = await slashingProposer.read.yeaCount([instanceAddress, roundNumber, info[1]]);
+      const bn = await t.ctx.cheatCodes.eth.blockNumber();
+      const slotNumber = await rollup.getSlotNumber();
```

Comment:
at some point we might want to multicall these, could there be a tiny tiny chance they cross boundaries

Reply:
These values only for debugging purposes e.g., logging if running the tests and they are failing, so I am not going to address that atm.